packages/vscode/src/extension.ts

```diff
@@ -62,7 +63,11 @@ function createLanguageServer(
 
 export async function activate(context: ExtensionContext): Promise<void> {
   const isDebugOrTest = isDebugOrTestSession()
-  const isTest = isDebugOrTest && !isDebugMode()
+
+  let rootPath = workspace.rootPath
+  if (rootPath) {
+    watcher = chokidar.watch(path.join(rootPath, 'node_modules/.prisma/client/index.d.ts'))
```

Comment:
`node_modules` might not be present at `project` root (in case of `lerna` etc)

Reply:
Good point, I'll change that to a glob pattern 👍